### PR TITLE
Limit data in the generated payload

### PR DESCRIPTION
**Pull request by ludeeus**
We only need to store the things we graph in the main object.
In case we want to change it up later, everything will be stored as `history:${timestampString}`

### CODE REVIEW

File: worker/src/handlers/schedule.ts [hunk 1]
```diff
@@ -22,9 +23,19 @@ export async function handleSchedule(event: ScheduledEvent): Promise<void> {
...
-  storedAnalytics[timestampString] = currentDataset;
+  const storeKey = `history:${timestampString}`;
 
-  await KV.put("core_analytics", JSON.stringify(storedAnalytics));
+  for (const key of Object.keys(storedAnalytics)) {
+    core_analytics[key] = {
+      active_installations: storedAnalytics[key].active_installations,
+      installation_types: storedAnalytics[key].installation_types,
+    };
+  }
```
Comment (bramkragten) [inline worker/src/handlers/schedule.ts:33]: Wouldn't it make more sense to turn this around (would then ofc also need site change):
```
active_installations: [{timestamp: number}, {timestamp: number}, {timestamp: number}, {timestamp: number}]
```

Comment (bramkragten) [reply worker/src/handlers/schedule.ts:33]: For a next PR, we should probably also send less old data, no need to send hourly data for a year back, that can just be daily?

**Review by bramkragten — APPROVED** [review]

File: worker/src/handlers/schedule.ts (hunk 1 above)
Comment (ludeeus) [reply worker/src/handlers/schedule.ts:33]: Want to eventually have years, months/weeks, days, and hours depending on how far back the date is.

Comment (ludeeus) [reply worker/src/handlers/schedule.ts:33]: > Wouldn't it make more sense to turn this around

_Maybe_ but the current structure is more true to the original datasets.

**Pull request merged**